src/node_crypto.cc

```diff
@@ -212,6 +212,115 @@ static int NoPasswordCallback(char* buf, int size, int rwflag, void* u) {
 }
 
 
+// namespace node::crypto::error
+namespace error {
+Local<Object> Decorate(Environment* env, Local<Object> obj,
+                             unsigned long err) {  // NOLINT(runtime/int)
+  if (err == 0) return obj;  // No decoration possible.
+
+  const char* ls = ERR_lib_error_string(err);
+  const char* fs = ERR_func_error_string(err);
+  const char* rs = ERR_reason_error_string(err);
+
+  Isolate* isolate = env->isolate();
+  Local<Context> context = isolate->GetCurrentContext();
+
+  if (ls != nullptr)
+    if (obj->Set(context, env->library_string(),
+        OneByteString(isolate, ls)).IsNothing())
```

Comment:
Can you line up the arguments here and elsewhere? For clarity I'd also use braces for branches that span more than one line.